go.mod

```diff
@@ -0,0 +1,13 @@
+module github.com/asecurityteam/component-stat
+
+go 1.12
+
+require (
+	github.com/asecurityteam/settings v0.1.0
+	github.com/go-yaml/yaml v2.1.0+incompatible // indirect
```

Comment:
task: I believe this needs to be `gopkg.in/yaml.v2` in all situations where we include the `yaml` library.